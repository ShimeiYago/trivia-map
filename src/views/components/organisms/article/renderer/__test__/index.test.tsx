import { shallow, ShallowWrapper } from 'enzyme';
import { Renderer, Props } from '..';

let wrapper: ShallowWrapper<Props, unknown, Renderer>;

const basicProps: Props = {
  title: 'title',
  content: 'content',
};

describe('Shallow Snapshot Tests', () => {
  beforeEach(() => {
    wrapper = shallow(<Renderer {...basicProps} />);
  });

  it('basic', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
