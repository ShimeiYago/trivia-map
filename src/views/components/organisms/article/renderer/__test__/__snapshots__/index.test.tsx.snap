// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Shallow Snapshot Tests basic 1`] = `
<Fragment>
  <ForwardRef(Typography)
    component="h2"
    variant="h6"
  >
    title
  </ForwardRef(Typography)>
  <ForwardRef(Typography)
    sx={
      Object {
        "mt": 2,
      }
    }
  >
    content
  </ForwardRef(Typography)>
</Fragment>
`;
